ui/src/components/SpotifyPlayer/index.tsx

```diff
@@ -82,15 +149,46 @@ const SpotifyPlayer = ({
       {loginToken && (
         <div>
           <section style={{ marginBottom: "75px" }}>
-            <img src={song?.album.images[0].url} />
+            <img
+              alt={`${song?.name} Album Art`}
+              src={song?.album.images[0].url}
+            />
           </section>
           <aside>
             <h5>{song?.name}</h5>
             <h5>{song?.artists.map((item) => item.name).join(", ")}</h5>
-            <h5>{isPlaying ? "Playing" : "Paused"}</h5>
+            <h5>{isPlaying.toString()}</h5>
             <div>
               <div style={progressBarStyles} />
             </div>
+            <div>
+              <IconButton
+                onClickArgs={PlayerCommand.Backwards}
+                onClick={executePlayerCommand}
```

Comment:
I don't think you need `onClickArgs` as a prop at all.
```suggestion
                onClick={() => executePlayerCommand(PlayerCommand.Backwards)}
```